srcpkgs/gogs/INSTALL

```diff
@@ -3,7 +3,7 @@
 case "$ACTION" in
 post)
 	if [ "$UPDATE" = "no" ]; then
-		chown gogs:gogs etc/gogs.ini
+		chown gogs:gogs /etc/gogs/conf.ini
```

Comment:
Paths in INSTALL scripts must be relative in case xbps-install is used with `--rootdir <PATH>` and `chroot(2)` isn't available.

Reply:
Done!